frontend/src/modules/lf/segments/components/view/lf-projects-table.vue

```diff
@@ -169,15 +169,15 @@
       </template>
     </el-table-column>
 
-    <template v-if="project.subprojects?.length && (hasPermissionToCreateSubProject && hasAccessToSegmentId(project.id))" #append>
+    <template v-if="project.subprojects?.length && (hasPerm && hasAccessToSegmentId(project.id))" #append>
       <div class="w-full flex justify-start p-6">
         <el-button class="btn btn-link btn-link--primary" @click="emit('onAddSubProject', project.slug)">
           + Add sub-project
         </el-button>
       </div>
     </template>
 
-    <template v-if="(hasPermissionToCreateSubProject && hasAccessToSegmentId(project.id))" #empty>
+    <template v-if="(hasPermission(LfPermission.subProjectCreate) && hasAccessToSegmentId(project.id))" #empty>
```

Comment:
For some reason "Add sub-project" is not showing up for admins, so this validation is failing